### ISSUE TITLE
'cvParameters' block is allowed only Character Variant (cvXX) features.

### ISSUE THREAD

**Issue by miguelsousa** [adobe-type-tools/afdko#237]
After applying the workaround in #236 attempting to build [Source Code Regular](https://github.com/adobe-fonts/source-code-pro/tree/master/Roman/Instances/Regular) fails with
```
makeotfexe [FATAL] <SourceCodePro-Regular> A 'cvParameters' block is allowed 
only Character Variant (cvXX) features. It is being used in feature 'cv10'.
```

The code for the `cv10` feature is [here](https://github.com/adobe-fonts/source-code-pro/blob/master/Roman/familyGSUB.fea#L469-L480) and it's structured like the previous cvXX features.

**Comment by cjchapman** [comment]
I've got a proposed fix for this here:
https://github.com/adobe-type-tools/afdko/pull/243

**Comment by miguelsousa** [comment]
Fixed by #243. afdko v2.6.25
